galician/dixital/dixital.md

```diff
@@ -5,3 +5,4 @@ A viaxe espacial a través do canal dixital comezara.
 A información recollida dos cúmulos estelares iniciaba unha nova era.
 
 [O asomarse](mortos/mortos.md)
+[Ao dar a volta](aire/aire.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.